include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3651,13 +3651,19 @@ struct OrtApi {
    *     - "73"
    *     - "75"
    *   "device_id": The ID of the device to use when setting 'htp_arch'. Defaults to "0" (for single device).
-       "enable_htp_fp16_precision": Used for float32 model for HTP backend.
-       Enable the float32 model to be inferenced with fp16 precision. Otherwise, it will be fp32 precision.
-         - "0": With fp32 precision.
-         - "1": Default. With fp16 precision.
-       "enable_htp_weight_sharing": Enable QNN weight sharing feature while compiling multiple graphs into one QNN context.
-         - "0": Default. Disabled.
-         - "1": Enabled.
+   *   "enable_htp_fp16_precision": Used for float32 model for HTP backend.
+   *   Enable the float32 model to be inferenced with fp16 precision. Otherwise, it will be fp32 precision.
+   *     - "0": With fp32 precision.
+   *     - "1": Default. With fp16 precision.
+   *   "enable_htp_weight_sharing": Enable QNN weight sharing feature while compiling multiple graphs into one QNN context.
+   *     - "0": Default. Disabled.
+   *     - "1": Enabled.
+   *   "offload_graph_input_quantization": Offload graph input quantization to another execution provider (typically CPU EP).
```

Comment:
Do we need to split input and output separately, or just use one flag offload_graph_io_quantization to handle them all?

Reply:
yea I think I agree with Hector. better to have 1 option,  unless there's cases (can't think of right now) where we only only want one and not the other.

Reply:
also does it need to be a QNN specific option

Reply:
I updated to one option.